lib/styles/modifiers/_colors.scss

```diff
@@ -27,7 +29,7 @@
 }
 
 .-fireflyBlack {
```

Comment:
IMO, it isn't enough to change color values here, we should also migrate class names. For this phase of migration I think we shouldn't touch this file

Reply:
Right.